mdapi/database/base.py

```diff
@@ -28,7 +28,7 @@
 
 
 def index_database(name, tempdtbs):
-    servlogr.logrobjc.info("[%s] Indexing database %s" % (name, tempdtbs))
```

Comment:
I wonder if the f-string changes can stay and work just fine with the logging implementation that I have in the project or if I should address the change according to the recommendation you made here. 🤔

Reply:
The reason why I would not use an f-string here – and I would use them just about everywhere else – is not that it won’t work fine (it will), but that it would always expand the placeholders in the format string, even if logging configuration makes the message not end up anywhere.

Letting the logger (or probably rather the formatter) do the work lazily, only if necessary, is a performance optimization. It’s admittedly awkward that it makes us use two different ways of string interpolation (new-style in f-strings, %-style in log messages), but I’d do it anyway – once it becomes a habit, it comes at no cost.